src/Illuminate/Auth/Access/Gate.php

```diff
@@ -162,7 +164,7 @@ public function check($ability, $arguments = [])
      *
      * @param  string  $ability
      * @param  array  $arguments
-     * @return callable
+     * @return mixed
```

Comment:
Sure. https://github.com/lucasmichot/framework/commit/2dc0908a7807af34ea36e83e07f00e2eb0d525f4
There might a some return statements to change in Broadcasting and Queue then ?
https://github.com/laravel/framework/blob/5.1/src/Illuminate/Queue/Jobs/Job.php#L135
Also https://github.com/laravel/framework/blob/5.1/src/Illuminate/Support/Str.php#L196 maybe too